test/route.test.js

```diff
@@ -65,7 +65,7 @@ test('route', t => {
     }
   })
 
-  test('invalid schema - route', t => {
+  test('invalid schema - route', { skip: true }, t => {
```

Comment:
@mcollina this test is failing:

before: the errors building schemas was evaluated in the avvio flow, and the error was the input to the `after`, so the user could ignore the error

now: the error is evaluated after all the avvio flow (in preready event) and the `after` doesn't receive anymore this test.

what do you think about this case?

I think that the `skip the error in after` is quite non-sense because it means that some routes didn't load as the user expect

Reply:
I would replace the check inside after with a check inside ready, wdyt?

Reply:
It was my first change to fix this test 👍 

Pushed